Gemfile

```diff
@@ -14,7 +14,7 @@ gem "minima", "~> 2.5"
 
 # If you want to use GitHub Pages, remove the "gem "jekyll"" above and
 # uncomment the line below. To upgrade, run `bundle update github-pages`.
-gem "github-pages", "~> 219", group: :jekyll_plugins
+gem "github-pages", "~> 232", group: :jekyll_plugins
```

Comment:
Do we need any version pinning at all?

Reply:
Agreed. But I was hesitant to do that, thinking there might be some benefit to pinning, given that our current code is pinned. @moble is it safe to use the latest version?